modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsRelationshipsDisplayContext.java

```diff
@@ -60,6 +64,9 @@ public ObjectDefinitionsRelationshipsDisplayContext(
 		_objectDefinitionService = objectDefinitionService;
 
 		_objectRequestHelper = new ObjectRequestHelper(httpServletRequest);
+
+		_systemObjectDefinitionMetadataTracker =
+			systemObjectDefinitionMetadataTracker;
```

Comment:
Hi @rodrigopaulino, I'm not sure about this, but I always thought that we should first initialize the variables that are constructor parameters, like in [here](https://github.com/carolmariaabb/liferay-portal/blob/d8e7a6953984d9dd7076f960fade724d5cbef404/modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContext.java#L146-L150).

Reply:
But since I'm not sure and it's something simple, we can forward this PR as it is if the CI passes.